app/controllers/integrations_controller.rb

```diff
@@ -0,0 +1,46 @@
+class IntegrationsController < ApplicationController
+  authorize_resource
+  before_action :load_project
+
+  respond_to :html, :json
+
+  def index
+    @integrations = @project.integrations
+    @integration = Integration.new
+    respond_with(@integrations)
```

Comment:
`@integrations` [aren't being used on view](https://github.com/Codeminer42/cm42-central/pull/3/files#diff-2212560b4714bca5146aa0e8fc2c0558R1), so you can drop the [assign line](https://github.com/Codeminer42/cm42-central/pull/3/files#diff-93b21e526fd2ba0e475e69f6ea52b46eR8) and use just `respond_with(@project.integrations)`